packages/medusa/src/api/middlewares/transformQuery.ts

```diff
@@ -0,0 +1,143 @@
+import { NextFunction, Request, Response } from "express"
```

Comment:
well catch :)

Reply:
@srindom let me know if I am wrong but at the moment the controllers does serve any other purpose than fetching simple data no ? since all the config is done before reaching the handler